Readme.md

```diff
@@ -28,6 +28,8 @@ Create a `Brewfile` in the root of your project:
 
 Then list your Homebrew based dependencies in your `Brewfile`:
 
+    cask_opts appdir: '/Applications'
```

Comment:
Sorry, I know I've changed my mind here but `cask_args` feels like it may be more consistent.